src/app/reference-data/reference-data.component.html

```diff
@@ -3,12 +3,11 @@ <h3>Reference Data</h3>
     <section id="ref-data-dropdown">
         <div class="dropdown" id="dropdown">
             <h5>Type:</h5>
-            <select class="form-select" aria-label="Default select example">
-                <option selected>Select Reference Data</option>
-                <option value="1">Cities</option>
-                <option value="2">Typology</option>
-                <option value="3">Realestate Type</option>
-                <option value="3">Amenities</option>
+            <select class="form-select" id="referenceDataDropDown" (change)="dropDownFilter()" [(ngModel)]="selectedRefDataType" aria-label="Default select example">
+                <option value="1">Typology</option>
+                <option value="2" selected>Realestate Type</option>
```

Comment:
I believe if you remove this "selected", next to value=2, it will work as well because you are setting the default selected value in typescript

Reply:
Done.